internal/cmd/schema-registry/command_cluster_upgrade.go

```diff
@@ -35,36 +39,57 @@ func (c *command) newClusterUpgradeCommand() *cobra.Command {
 }
 
 func (c *command) clusterUpgrade(cmd *cobra.Command, _ []string) error {
-	packageDisplayName, err := cmd.Flags().GetString("package")
+	environmentId, err := c.Context.EnvironmentId()
 	if err != nil {
 		return err
 	}
 
-	packageInternalName, err := getPackageInternalName(packageDisplayName)
+	clusters, err := c.V2Client.GetSchemaRegistryClustersByEnvironment(environmentId)
 	if err != nil {
 		return err
 	}
 
-	environmentId, err := c.Context.EnvironmentId()
+	if len(clusters) == 0 {
+		return errors.NewSRNotEnabledError()
+	}
+
+	cluster := clusters[0]
+	clusterSpec := cluster.GetSpec()
+
+	packageDisplayName, err := cmd.Flags().GetString("package")
 	if err != nil {
 		return err
 	}
 
-	cluster, err := c.Context.FetchSchemaRegistryByEnvironmentId(environmentId)
+	_, err = getPackageInternalName(packageDisplayName)
 	if err != nil {
 		return err
 	}
 
-	if packageInternalName == cluster.Package {
+	if strings.EqualFold(clusterSpec.GetPackage(), packageDisplayName) {
```

Comment:
Should we upper case these strings, and then do a regular `==` comparison? I don't think many people know what `EqualFold()` does (I had to look it up)